src/utilities/Api.js

```diff
@@ -0,0 +1,62 @@
+const API_URL = import.meta.env.VITE_API_URL;
+
+// fetch for categories
+export const getCategories = async () => {
+  const response = await fetch(`${API_URL}/categories"`);
+  return response.json();
+};
+
+export const addCategory = async (newCategory) => {
+  const response = await fetch(`${API_URL}/categories"`, {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify(newCategory),
+  });
+  return response.json();
+};
+
+export const editCategory = async (categoryId, categoryEdit) => {
+  const response = await fetch(`${API_URL}/categories/${categoryId}`, {
+    method: "PATCH",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify(categoryId, categoryEdit),
+  });
+  return response.json();
+};
+
+export const deleteCategory = async (categoryId) => {
+  await fetch(`${API_URL}/categories/${categoryId}`, {
+    method: "DELETE",
```

Comment:
This route will return an status code from deletions. Return this status code for the moment.

Reply:
I assigned the fetch to a response variable, then I returned the response.status. I think this should work?